activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -151,7 +151,12 @@ def extract_callstack(callstack)
         end
 
         def _extract_callstack(callstack)
-          warn "Please pass `caller_locations` to the deprecation API" if $VERBOSE
+          ActiveSupport.deprecator.warn(<<~MESSAGE)
+            Passing the result of `caller` to ActiveSupport::Deprecation#warn is deprecated and will be removed in Rails 7.2.
```

Comment:
Is 7.2 the correct version here?

Reply:
Nope, you are right 🤦

Reply:
Fixed in https://github.com/rails/rails/commit/514a90343195e5703e3aee6cc582aadcc7dda632